server/src/main/java/org/elasticsearch/action/admin/indices/diskusage/IndexDiskUsageAnalyzer.java

```diff
@@ -289,6 +289,17 @@ void analyzeDocValues(SegmentReader reader, IndexDiskUsageStats stats) throws IO
                     throw new IllegalStateException("Unknown docValues type [" + dvType + "]");
                 }
             }
+            switch (field.docValuesSkipIndexType()) {
+                case NONE -> {
+                }
+                case RANGE -> {
+                    var skipper = docValuesReader.getSkipper(field);
+                    while (skipper.maxDocID(0) != DocIdSetIterator.NO_MORE_DOCS) {
```

Comment:
I think we can use the document after the max doc of the highest level (`skipper.maxDocID(skipper.numLevels() - 1) + 1`) to reduce the number of iterations and bytes read.